src/components/settings-menu/components/MenuItem.tsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import { Source } from 'react-native-fast-image';
+import { ButtonPressAnimation } from '../../animations';
+import CheckmarkCircledIcon from '../../icons/svg/CheckmarkCircledIcon';
+import WarningIcon from '../../icons/svg/WarningIcon';
+import Caret from '@rainbow-me/assets/family-dropdown-arrow.png';
+import { Box, Inline, Stack, Text } from '@rainbow-me/design-system';
+import { ImgixImage } from '@rainbow-me/images';
+import { useTheme } from '@rainbow-me/theme';
+
+const ICON_SIZE = 60;
+
+interface ImageIconProps {
+  source: StaticImageData;
+}
+
+const ImageIcon = ({ source }: ImageIconProps) => (
+  <Box
+    as={ImgixImage}
+    borderRadius={ICON_SIZE / 2}
+    height={{ custom: ICON_SIZE }}
+    marginLeft={{ custom: -11 }}
+    marginRight={{ custom: -11 }}
+    marginTop={{ custom: 8 }}
+    source={source as Source}
+    width={{ custom: ICON_SIZE }}
+  />
+);
+
+interface SelectionProps {
+  children: React.ReactNode;
+}
+
+const Selection = ({ children }: SelectionProps) => (
+  <Text color="secondary60" size="18px" weight="semibold">
+    {children}
+  </Text>
+);
+
+export enum StatusType {
+  Complete = 'complete',
+  Incomplete = 'incomplete',
+  Warning = 'warning',
+  Selected = 'selected',
+}
+
+interface StatusIconProps {
+  status: StatusType;
+}
+
+const StatusIcon = ({ status }: StatusIconProps) => {
+  const { colors, isDarkMode } = useTheme();
+  let color;
+  switch (status) {
+    case StatusType.Complete:
+      color = colors.green;
+      break;
+    case StatusType.Incomplete:
+      color = colors.alpha(colors.blueGreyDark, 0.5);
+      break;
+    case StatusType.Warning:
+      color = colors.orangeLight;
+      break;
+    case StatusType.Selected:
+      color = colors.appleBlue;
+      break;
+    default:
+      break;
+  }
+  return (
+    <Box
+      as={status === StatusType.Warning ? WarningIcon : CheckmarkCircledIcon}
+      backgroundColor={color}
+      color={color}
+      colors={colors}
+      fillColor={colors.white}
+      shadowColor={isDarkMode ? colors.shadow : color}
+      shadowOffset={{
+        height: 4,
+        width: 0,
+      }}
+      shadowOpacity={0.4}
+      shadowRadius={6}
+    />
+  );
+};
+
+interface TitleProps {
+  text: string;
+  weight?: 'regular' | 'medium' | 'semibold' | 'bold' | 'heavy';
+  disabled?: boolean;
+  isLink?: boolean;
+  colorOverride?: string;
+}
+
+const Title = ({
+  text,
+  weight = 'semibold',
+  disabled,
+  isLink,
+  colorOverride,
+}: TitleProps) => (
+  <Text
+    color={
+      colorOverride
+        ? { custom: colorOverride }
+        : disabled
+        ? 'secondary60'
+        : isLink
+        ? 'action'
+        : 'primary'
+    }
+    containsEmoji
+    size="18px"
+    weight={weight}
+  >
+    {text}
```

Comment:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/6843656/180036761-74e372d0-e36a-4142-838a-ba2fd79a98cd.png">

Reply:
one small nit here, this ends up moving to the second line instead of the text breaking into 2 lines and being vertically centered with the emoji.

tried to fix it but couldn't figure out how so maybe you know